### ISSUE TITLE
Signing with contract wallet may generate signatures longer than 64/65 bytes

### ISSUE THREAD

**Issue by mingyan-space** [ProjectOpenSea/seaport-js#323]
### Component

API

### Have you ensured that all of these are up to date?

- [X] seaport-js
- [X] Node (minimum v16)

### What version of seaport-js are you on?

v2.0.5

### What function is the bug in?

signOrder

### Operating System

macOS (Apple Silicon)

### Describe the bug

When testing signOrder with contract wallets (including Safe, Unipass), they may return much longer signature. This can throw an error with `ethers.utils.splitSignature(signature).compact`. 

```
public async signOrder(
    orderComponents: OrderComponents,
    accountAddress?: string,
  ): Promise<string> {
    const signer = this._getSigner(accountAddress);

    const domainData = await this._getDomainData();

    const signature = await signer._signTypedData(
      domainData,
      EIP_712_ORDER_TYPE,
      orderComponents,
    );

    // Use EIP-2098 compact signatures to save gas.
    return ethers.utils.splitSignature(signature).compact;
  }
```

Is there a way to bypass this compact procedure? 
Thank you

**Comment by ryanio** [comment]
ah i see, thanks for reporting, opening a PR targeting to fix now

**Comment by ryanio** [comment]
released in [v2.0.6](https://github.com/ProjectOpenSea/seaport-js/releases/tag/v2.0.6)
